doc/Example-Gradle-Project/build.gradle

```diff
@@ -0,0 +1,146 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = 'org.sosy_lab.java_smt_example.SolverOverviewTable'
+
+repositories {
+  // try to resolve by pom first
+  mavenCentral {
+    metadataSources {
+      mavenPom()
+    }
+  }
+    // try to get artifacts if no useful pom is found
+  mavenCentral {
+    metadataSources {
+      artifact()
+    }
+  }
+    // Ivy can be used as well
+  ivy {
+    url "https://www.sosy-lab.org/ivy"
+    patternLayout {
+      artifact "/[organisation]/[module]/[classifier]-[revision].[ext]"
+      //ivy "/[organisation]/[module]/ivy-[revision].xml"
+    }
+    metadataSources {
+       artifact()
+    }
+  }
+  // SMT-Solver Princess repo
```

Comment:
Is this repository still used? Princess switched to a puiblic Maven repo some time ago.

Reply:
Seems like it as it has the newest version, but i've switched to Maven central for Princess in the latest commit.